os_migrate/plugins/modules/import_workload_dst_failure_cleanup.py

```diff
@@ -231,9 +237,10 @@ def _delete_volumes(self):
 
 def run_module():
     argument_spec = openstack_full_argument_spec(
+        auth=dict(type='dict', no_log=True, required=True),
         data=dict(type='dict', required=True),
         conversion_host=dict(type='dict', required=True),
-        ssh_key_path=dict(type='str', required=True),
+        ssh_key_path=dict(type='str', required=True, no_log=True),
```

Comment:
It's just a path, not the actual content of the key. I don't think this should be no_log, we may want this as debugging info, but it should not cause any security concerns even when logged IMO.

Reply:
Now reverted in https://github.com/os-migrate/os-migrate/pull/484/commits/dcb6ba052d85e887f34570d66b489a6b7a0a09da